minimint/Cargo.toml

```diff
@@ -41,6 +41,4 @@ tokio = { version = "1.0.1", features = ["full"] }
 tokio-rustls = "0.23.4"
 tokio-util = { version = "0.6.0", features = [ "codec" ] }
 tracing ="0.1.22"
-tracing-opentelemetry = "0.17.2"
 tracing-subscriber = { version = "0.3.1", features = [ "env-filter" ] }
-x509-parser = "0.14.0"
```

Comment:
Uff, that was my mistake. We should make `cargo udeps` part of CI to avoid that in the future imo.